doc_examle_test.go

```diff
@@ -0,0 +1,115 @@
+package comver_test
```

Comment:
@tangrufus This files could be renamed to `doc_example_test.go` (note the p)

Reply:
Good catch. I need ☕ 

Please don't use this package for now. I am not satisfied with the API yet.
API changes are coming

Reply:
Fixed in #12

Thanks!